### PR TITLE
Minor updates to CWP policies

### PR DESCRIPTION
**Pull request by BenLloydPearson**
#### Summary
This is part of some slight modifications to the community writing program intake process.

### CODE REVIEW

File: contributors/contributors/ways-to-contribute/community-content-program.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ By submitting your application to the Mattermost Community Writing Program you a
...
+The Mattermost blog produces original, first-run content that has never been published previously on another website. All code, copy, images, and other content should be the result of the author's original thought, work, research, and self-expression. At our discretion, Mattermost will cease all work, possibly without notice, with any author that reproduces someone else's work to claim as their own or attempts to submit content that has already been published somewhere else. This includes borrowing content, wording, or organizational structure from another person, organization, website, published articles, code, documentation, or any other content resources.
```
Comment (jordanafung) [inline contributors/contributors/ways-to-contribute/community-content-program.md]:
```suggestion
The Mattermost blog produces original, first-run content that has never been published previously on another website. All code, copy, images, and other content should be the result of the author's original thought, work, research, and self-expression. At our discretion, Mattermost will cease all work, possibly without notice, with any author that reproduces someone else's work to claim as their own or that attempts to submit content that has already been published somewhere else without permission and discussions with our team. This includes borrowing content, wording, or organizational structure from another person, organization, website, published articles, code, documentation, or any other content resources.
```

Comment (jordanafung) [reply contributors/contributors/ways-to-contribute/community-content-program.md]: Added addendum for when they have published on their own blog but for example creating an updated post for us.
Feel free to decline if we are trying to be stricter though/if we think things will muddy the water

**Review by azigler — DISMISSED** [review]
@BenLloydPearson @jordanafung Thanks y'all! I tightened up the language in some areas, based on policies I've seen and written in the past. This looks good to me!

**Comment by BenLloydPearson** [comment]
@azigler I updated the content of this based on our conversation today.

**Review by azigler — DISMISSED** [review]
Looks great @BenLloydPearson! Thanks for revising.

File: contributors/contributors/ways-to-contribute/community-content-program.md [hunk 2]
```diff
@@ -51,6 +51,9 @@ Our editorial staff will reach out to let you know if we want to publish your co
 
 Our team reviews every submission, and we look for technical content that comprehensively covers software development topics with clear explanations. Explain as much detail as possible and leave no concepts out.
 
+**When can I expect to hear about my submission?**
+Our topic review cycles happen once every three months, in the 2nd half of the months of March, June, September, and December. In most cases, you will first hear from us following one of these review periods. However, we sometimes make exceptions on a case-by-case basis as proposals are submitted to us, so we sometimes reach out to authors outside of the official review periods.
```
Comment (jordanafung) [inline contributors/contributors/ways-to-contribute/community-content-program.md:55]:
```suggestion
Our topic review cycles happen once every three months, in the 2nd half of the months of March, June, September, and December. In most cases, you will first hear from us following one of these review periods. However, depending on bandwidth, we sometimes make exceptions on a case-by-case basis as proposals are submitted to us, so we sometimes reach out to authors outside of the official review periods.
```

File: contributors/contributors/ways-to-contribute/community-content-program.md [hunk 3]
```diff
@@ -71,15 +74,15 @@ By submitting your application to the Mattermost Community Writing Program you a
...
+The Mattermost blog produces original, first-run content that has never been published previously in any form, digital or physical. All code, copy, images, and other content should be the result of the author's original thought, work, research, and self-expression. At our discretion, Mattermost will cease all work — possibly without notice — with any author that reproduces, copies, or claims someone else's work as their own. This policy extends to content that has been published, or is in the process of being published, by the author elsewhere without prior permission from Mattermost. This policy includes borrowing content, wording, code, documentation, or organizational structure from any other person or organization's prior work.
```
Comment (jordanafung) [inline contributors/contributors/ways-to-contribute/community-content-program.md:81]:
```suggestion
The Mattermost blog produces original, first-run content that has never been published previously in any form, digital or physical. All code, copy, images, and other content should be the result of the author's original thought, work, research, and self-expression. At our discretion, Mattermost will cease all work — possibly without notice — with any author that reproduces, copies, or claims someone else's work as their own. This policy extends to content that the author has published, or is in the process of publishing, elsewhere without prior permission from Mattermost. This policy includes borrowing content, wording, code, documentation, or organizational structure from any other person or organization's prior work.
```

**Comment by jordanafung** [comment]
Thanks Ben! This looks great.
I added small comments to increase clarity.

**Review by azigler — APPROVED** [review]
Thanks for adding that, @BenLloydPearson!

**Pull request merged**